project/dimensions/subset/subsectors_by_sector.csv

```diff
@@ -14,12 +14,15 @@ bev_vocational_truck,,,,x
 bev_heavy_freight_truck,,,,x
 bev_bus,,,,x
 rail_transit,,,,x
-naics_3273,,x,,
-naics_3253,,x,,
-naics_3311,,x,,
-naics_311_312,,x,,
-naics_325_other,,x,,
-naics_3241,,x,,
-naics_322,,x,,
-naics_3_other,,x,,
-data_center,x,,,
+cement_concrete,,x,,
+chemical,,x,,
+construction,,x,,
+datacenter,,x,,
+food_bev,,x,,
+iron_steel,,x,,
+mining,,x,,
+other,,x,,
+pesticides_fertilizer,,x,,
+pet_refinery,,x,,
+pulp_paper,,x,,
+water_treatment,,x,,

```

Comment:
Even if it's not so indicated in the original dataset, maybe we should map datacenter and water_treatment to com instead of to ind here. That could be helpful for automating residual calculations?